website/settings/defaults.py

```diff
@@ -174,6 +176,15 @@ def parent_dir(path):
 ## Default RabbitMQ broker
 BROKER_URL = 'amqp://'
 
+CELERY_CREATE_MISSING_QUEUES = False
+CELERY_DEFAULT_QUEUE = 'osf'
+CELERY_QUEUES = (
```

Comment:
The celery docs advise against overriding this in the common case. Can you add a comment explaining this?